homeassistant/components/homematicip_cloud/quality_scale.yaml

```diff
@@ -0,0 +1,3 @@
+rules:
```

Comment:
Please make a separate PR for the quality scale update.

Reply:
Removed the update of quality scale, but left the empty file